### PR TITLE
Feature/polishing css search

### PR DESCRIPTION
**Pull request by joanafbrito**
# Description drawing

- Polish to css Pokemon details  to match the Pokedex Cards in the grid( still have to polish the text)
- Polish img size on Pokedex grid  ( and grid container suggestion because how it was looking in my computer)- changed from px to %
- Add a quote do the How to page ( end of the page) 
- Add style to the search bar 
- Fixed an error with the x button ( it was still showing the error message )
- Add some style to the error message ( for the input)
- Renamed the Help component to be a HowTo  component for developer empahty

Next steps:

Fix sizing of the text in the PokemonDetails to be responsive.
Click the card and make it go to the `PokemonDetails` page
Have an x button for the `PokemonDetails` Page to go back to the `PokedexGrid` page

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/App/App.css [hunk 1]
```diff
@@ -3,6 +3,15 @@
   display: flex;
   flex-direction: column;
 }
+
+.search-error-message {
+  font-family: 'PocketMonk';
+  font-size: 35px;
+  text-align: center;
+  color: #ffeb3b91;
+  -webkit-text-stroke: 0.5px #3b4cca;
```
Comment (Matt-Roden) [inline src/components/App/App.css:12]: Nice little touch here for the user!

File: src/components/App/App.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import Search from "../Search/Search";
 import { getPokedexData } from "../../apiCalls";
 import PokedexGrid from "../PokedexGrid/PokedexGrid";
 import Header from "../Header/Header";
-import Help from "../Help/Help";
+import HowTo from "../HowTo/HowTo";
```
Comment (Matt-Roden) [inline src/components/App/App.js:6]: Thanks for making this more empathetic for the developers

Comment (hayleyw7) [inline src/components/App/App.js:6]: Thanks for the rename. Makes a lot more sense!

File: src/components/App/App.js [hunk 3]
```diff
@@ -70,12 +70,10 @@ class App extends Component {
   }
 
   clearPokemon = (e) => {
-    // e.preventDefault();
-    console.log('before clear', this.state.foundPokemon)
     this.setState({
       foundPokemon: [],
     });
-    console.log('before clear', this.state.foundPokemon)
+    this.clearErrorMessage();
```
Comment (hayleyw7) [inline src/components/App/App.js:76]: Good call. Good fix!

File: src/components/Header/Header.css [hunk 4]
```diff
@@ -22,9 +22,24 @@
...
   color: white;
+  /* color:#f2b32e; */
   text-decoration: none;
+  -webkit-text-stroke: 2px #3b4cca;
+  transition: 300ms;
+  animation: vert-bounce 4.5s infinite linear;
+  position: relative;
+}
+
+@keyframes vert-bounce {
```
Comment (hayleyw7) [inline src/components/Header/Header.css:35]: This is so fun.

File: src/components/HowTo/HowTo.js [hunk 5]
```diff
@@ -59,9 +59,13 @@ const Help = (props) => {
           </p>
 
         </article>
+        <section className='meowth-quote'>
```
Comment (hayleyw7) [inline src/components/HowTo/HowTo.js:62]: Fun thing to add! We just have to change the name to "Mew" instead of "Meowth".

**Review by Matt-Roden — APPROVED** [review]
Great work! looks great :)

File: src/components/HowTo/HowTo.js [hunk 6]
```diff
@@ -59,9 +59,13 @@ const Help = (props) => {
...
+           <img src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/151.svg" className="meowth-pic" alt="mew image"/>
```
Comment (hayleyw7) [inline src/components/HowTo/HowTo.js:63]: Thoughts on animating this image so that it flies like Mew?

**Review by hayleyw7 — APPROVED** [review]
Amazing!

**Pull request merged**